docker/aarch64/app/client_publish.py

```diff
@@ -0,0 +1,72 @@
+###############################################################################
+#
+# The MIT License (MIT)
+#
+# Copyright (c) Crossbar.io Technologies GmbH
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+###############################################################################
+
+from __future__ import print_function
+from os import environ
+import os
+import argparse
+from twisted.internet.defer import inlineCallbacks
+
+from autobahn.twisted.util import sleep
+from autobahn.twisted.wamp import ApplicationSession, ApplicationRunner
+
+
+class ClientSession(ApplicationSession):
+    """
+    An application component that publishes an event every second.
+    """
+
+    @inlineCallbacks
+    def onJoin(self, details):
+        print("session attached")
+        counter = 0
+        while True:
+            print('backend publishing com.myapp.hello', counter)
+            self.publish(u'com.myapp.hello', "Hello World %d"%counter)
+            counter += 1
+            yield sleep(1)
+
+
+if __name__ == '__main__':
+    import six
+    url = os.environ.get('CBURL', u'ws://localhost:8080/ws')
+    realm = os.environ.get('CBREALM', u'realm1')
+
+    # parse command line parameters
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-d', '--debug', action='store_true', help='Enable debug output.')
+    parser.add_argument('--url', dest='url', type=six.text_type, default=url, help='The router URL (default: "ws://localhost:8080/ws").')
+    parser.add_argument('--realm', dest='realm', type=six.text_type, default=realm, help='The realm to join (default: "realm1").')
+
+    args = parser.parse_args()
+    # any extra info we want to forward to our ClientSession (in self.config.extra)
+    extra = {
+        u'foobar': u'A custom value'
+    }
+
+    runner = ApplicationRunner(url=args.url, realm=args.realm, extra=extra)
+    runner.run(ClientSession, auto_reconnect=True)
+
```

Comment:
* realm name: Yes sure,I used realm1 because the default [crossbar  config.json ](https://github.com/crossbario/crossbar/blob/master/crossbar/node/templates/default/.crossbar/config.json)uses it, also it gets packed in the docker.  In the examples, It is any way  passed as parameter so we can set any realm name via command line, I will change the  getting-started doc accordingly. 
If it would not have much impact isn't it better to change in crossbar config.json too?
* Component API: Was not aware of it.Do you mean something like this [example](https://github.com/crossbario/autobahn-python/blob/master/examples/twisted/wamp/component/frontend_scram.py) , can you give me some other example that uses component example?